addons/test_mail_full/security/ir.model.access.csv

```diff
@@ -5,3 +5,5 @@ access_mail_test_sms_bl_all,mail.test.sms.bl.all,model_mail_test_sms_bl,,0,0,0,0
 access_mail_test_sms_bl_user,mail.test.sms.bl.user,model_mail_test_sms_bl,base.group_user,1,1,1,1
 access_mail_test_sms_partner_all,mail.test.sms.partner.all,model_mail_test_sms_partner,,0,0,0,0
 access_mail_test_sms_partner_user,mail.test.sms.partner.user,model_mail_test_sms_partner,base.group_user,1,1,1,1
+access_mail_test_portal_all,mail.test.portal.all,model_mail_test_portal,,1,0,0,0
```

Comment:
Could be defined before mail_test_sms_... lines (to keep alphabetical ordering when possible)

Reply:
Done